model-engine/model_engine_server/inference/forwarding/forwarding.py

```diff
@@ -142,7 +148,49 @@ class Forwarder(ModelEngineSerializationMixin):
     serialize_results_as_string: bool
     wrap_response: bool
     forward_http_status: bool
-    post_inference_hooks_handler: PostInferenceHooksHandler
+    post_inference_hooks_handler: Optional[PostInferenceHooksHandler] = None
+
+    async def forward(self, json_payload: Any) -> Any:
```

Comment:
This is a async version of __call__. I didn't want to refactor all uses of __call__ so i'm just creating a separate function to do this

Reply:
rename to `aforward` or `async_forward` to distinguish?

Reply:
eh, is prefixing w/ `a` standard practice? I feel like we should just default to async

Reply:
not really, mostly because there's already one function that's sync, and i want to make it clear about the differences